src/SourceBuild/content/repo-projects/roslyn-analyzers.proj

```diff
@@ -1,18 +1,15 @@
-<Project>
-  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <BuildCommandArgs>$(StandardSourceBuildArgs)</BuildCommandArgs>
-    <BuildCommandArgs>$(BuildCommandArgs) $(FlagParameterPrefix)warnAsError $(ArcadeFalseBoolBuildArg)</BuildCommandArgs>
-    <BuildCommand>$(StandardSourceBuildCommand) $(BuildCommandArgs)</BuildCommand>
-
     <LogVerbosityOptOut>true</LogVerbosityOptOut>
+
+    <BuildArgs>$(BuildArgs) $(FlagParameterPrefix)warnAsError $(ArcadeFalseBoolBuildArg)</BuildArgs>
   </PropertyGroup>
 
   <ItemGroup>
+    <RepositoryReference Include="arcade" />
     <RepositoryReference Include="runtime" />
```

Comment:
I'm not sure runtime is actually a live dependency...but that can be an issue for later.

Reply:
Yeah, but unrelated to this PR. I will clean this up in a follow-up though.

Reply:
Discussed with Matt offline. This should be evaluated as part of the parallel build VMR work.